addons/test_mail_sms/tests/test_sms_performance.py

```diff
@@ -36,7 +36,7 @@ def setUpClass(cls):
     def test_message_sms_record_1_partner(self):
         record = self.test_record.with_user(self.env.user)
         pids = self.customer.ids
-        with self.subTest("QueryCount"), self.mockSMSGateway(sms_allow_unlink=True), self.assertQueryCount(employee=27):  # tms: 27
+        with self.subTest("QueryCount"), self.mockSMSGateway(sms_allow_unlink=True), self.assertQueryCount(employee=31):  # tms: 31
```

Comment:
4 queries :o ? Why so much ?

Reply:
For triggering of the CRON job that needs to resolve the `ref` too, when cached, it is just 1 query.